index.html

```diff
@@ -8,21 +8,21 @@
       <link rel="stylesheet" href="style.css">
    </head>
    <body>
-      <article class="land">
+      <article id="land" class="land">
          <b>
          <header class="main-1">
             <a href="index.html" class="my-logo">George</a>
 
-            <nav>
+            <nav class="nav">
                 <ul class="my-menu-nav">
-                    <li class="menu-expand"><a href="#">Portfolio</a></li>
-                    <li class="menu-expand"><a href="#">About</a></li>
-                    <li class="menu-expand"><a href="#">Contact</a></li>
+                    <li class="menu-expand"><a href="#land" class="nav-link">Portfolio</a></li>
+                    <li class="menu-expand"><a href="#about" class="nav-link">About</a></li>
+                    <li class="menu-expand"><a href="#contact" class="nav-link">Contact</a></li>
```

Comment:
- [ ] Hey @george827 you did a good job trying to fix the mobile menu as requested by the previous reviewer. 👏 But there is another issue: the menu does not display just the blue background and the close icon. Please make sure to fix that, so that once the user clicks on the hamburger menu, the menu should appear. 👌

Your design 👇️
![Screenshot from 2022-10-17 21-48-58](https://user-images.githubusercontent.com/89228743/196291266-cade7c4a-6db4-436c-834c-429901712f59.png)

the Figma design 👇️
![Screenshot from 2022-10-17 21-57-22](https://user-images.githubusercontent.com/89228743/196291563-e7cb8d05-73ce-4952-9ad3-e7f822187fa9.png)